copyvios/checker.py

```diff
@@ -144,6 +144,10 @@ def _perform_check(query, page, use_engine, use_links):
             _LOGGER.exception("Failed to retrieve cached results")
 
     if not query.result:
+        if use_engine and not query.requester_username:
```

Comment:
I think we also need to check that this is not an API request.

Reply:
Done; added that extra condition.